corehq/form_processor/backends/couch/dbaccessors.py

```diff
@@ -126,6 +126,10 @@ def get_cases(case_ids, ordered=False):
         ]
 
     @staticmethod
+    def case_exists(case_id):
+        return CommCareCase.get_db().doc_exist(case_id)
```

Comment:
This is a broader check since it's not hitting a view that only includes cases. I think that's OK since we shouldn't have any docs with duplicate ID's anyway.